src/isomorphic/modern/class/__tests__/ReactClassEquivalence-test.js

```diff
@@ -32,7 +32,7 @@ describe('ReactClassEquivalence', function() {
 function runJest(testFile) {
   var cwd = process.cwd();
   var jestBin = path.resolve('node_modules', '.bin', 'jest');
-  var setupFile = path.resolve(__dirname, 'setupSpecEquivalenceReporter.js');
+  var setupFile = path.resolve(__dirname, 'setupSpecEquivalenceReporter.js.setup');
```

Comment:
What's the non-quick fix? Can we just do that? (I know how this works and we'll never come back to that)